tools/mocha-runner/package.json

```diff
@@ -21,6 +21,9 @@
       "output": [
         "bin/**",
         "tsconfig.tsbuildinfo"
+      ],
+      "dependencies": [
```

Comment:
why does it depend on puppeteer-core?

Reply:
Because of the Deflake action, if you do a full clean and build this part will fail.